script/helperFunctions.sh

```diff
@@ -2882,7 +2882,110 @@ function getCreate3FactoryAddress() {
 
   echo $CREATE3_FACTORY
 }
-  
+
+function convertToBcInt() {
+  echo "$1" | tr -d '\n' | bc
+}
+
+
+# transfers ownership of the given contract from old wallet to new wallet (e.g. new tester wallet)
+# will fail if old wallet is not owner
+# will transfer native funds from new owner to old owner, if old wallet has insufficient funds
+# will send all remaining native funds from old owner to new owner after ownership transfer
+transferContractOwnership() {
+    local PRIV_KEY_OLD_OWNER="$1"
+    local PRIV_KEY_NEW_OWNER="$2"
+    local CONTRACT_ADDRESS="$3"
+    local NETWORK="$4"
+
+    # Define minimum native balance
+    local MIN_NATIVE_BALANCE=$(convertToBcInt "100000000000000") # 100,000 Gwei
```

Comment:
What does `Bc` stand for?

Reply:
it's a tool to deal with bigger integers